pkg/controller/controller.go

```diff
@@ -97,10 +97,8 @@ func (c *Controller) Start(stopCh <-chan struct{}) error {
 		return nil
 	}
 
-	if c.bpfConfig.EnableBpfLog {
-		if err := logger.StartRingBufReader(ctx, c.mode, c.bpfConfig.BpfFsPath); err != nil {
-			return fmt.Errorf("failed to start ringbuf reader: %v", err)
-		}
+	if err := logger.StartRingBufReader(ctx, c.mode, c.bpfConfig.BpfFsPath); err != nil {
+		return fmt.Errorf("fail to start ringbuf reader: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to start ringbuf reader: %v", err)
```

So  we now use `kmeshctl log` to replace the function of `--enable-bpf-log` flag?

Reply:
No. 
This parameter was originally only used to create the ringbuf for `bpf_log`, and would actually cause an error if ringbuf was created. Not relevant for kmeshctl log.

Reply:
No. The two are not related.
Remove `enable-bpf-log` because if you don't create a ringbuf, it will cause a ringbuf blocking error.